stage2/modules/web-sockets/README.md

```diff
@@ -0,0 +1,52 @@
+# [Stage#2.](../../) WebSockets 🌐
+
+## Module Overview 📚
+
+The "WebSockets" module is crafted to provide an in-depth exploration of using WebSockets within the context of frontend web development. This module covers the essentials of WebSocket technology, its advantages over traditional HTTP communication, and how to implement WebSockets in a JavaScript environment. It aims to equip students with the skills needed to create real-time, bi-directional communication applications, such as chat applications, live notifications, and more.
+
+## Learning Objectives 🎯
+
+Students will:
+
+- Understand the core principles and advantages of WebSocket technology.
+- Learn how to establish a WebSocket connection using JavaScript.
+- Develop skills for sending and receiving messages through WebSockets.
+- Explore best practices for securing WebSocket connections.
+- Gain practical experience by building a simple real-time chat application.
+
+## Approximate Module Completion Time ⏱️
+
+- **4 hours**
+
+## Theory 📖
+
+Students are encouraged to explore the following resources:
+
+1. **Introduction to WebSockets:**
+
+   - [What are WebSockets?](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API)
+   - [Understanding WebSockets](https://web.dev/articles/websockets-basics)
+   - [WebSocket](https://javascript.info/websocket)
+
+2. **Implementing WebSockets in JavaScript:**
+
+   - [Creating a WebSocket chat application with JavaScript](https://www.tutorialspoint.com/websockets/websockets_quick_guide.htm)
+
+3. **Securing WebSocket Connections:**
+
+   - [Securing Your WebSocket Connections](https://devcenter.heroku.com/articles/websocket-security)
+   - [How to secure your WebSocket connections](https://www.freecodecamp.org/news/how-to-secure-your-websocket-connections-d0be0996c556/)
+
+4. **Lecture Videos:**
+   - [WebSockets [RU]](https://youtu.be/rpyk9JGAPZs)
+   - [Rolling Scopes School Lithuania. WebSocket](https://youtu.be/II77rlvVx-k)
+   - [Rolling Scopes School Georgia. WebSocket](https://youtu.be/kV5tOp7tdzM)
+
+## Practice 💻
+
+- Complete the "[St2] WebSockets" test in the RS-App > Auto Test.
```

Comment:
Shouldn't the [Fun Chat](https://github.com/rolling-scopes-school/tasks/blob/master/stage2/tasks/fun-chat/README.md) task also be included here?

Reply:
nope